### PR TITLE
[Wfw-8693wpt1w] integrate auth

### PR DESCRIPTION
**Pull request by LyubovSugak**
Auth0 was integrated, but i commented out AuthGuard otherwise rest of the routes will be not available on Netlify. Additional work within integration Auth0 and Netlify should be done (i'll create a ticket).

To run locally App with Auth0:
To make work Auth0 .env file should be created based on the file .env.sample. Keys ask on the FE slack channel
(FE onboarding doc was updated.)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +25,4 @@ yarn-error.log*
 
 .idea/
 .vscode/
+.env
```
Comment (Svitlana-G) [inline .gitignore:28]: Why did you add `.env` two times?

File: src/components/firstProfile/NameProfile.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const FULLNAME_REGEX = /^[a-zA-Zа-яА-ЯёЁ\s\p{L}]{2,15}$/u
 const NameProfile = () => {
   const { classes } = useStyles()
   const commonClasses = commonStyles().classes
-  const [fullName, setFullName] = useState('')
+  const [, setFullName] = useState('')
```
Comment (Svitlana-G) [inline src/components/firstProfile/NameProfile.tsx:19]: Why did you remove fullName?

Comment (LyubovSugak) [reply src/components/firstProfile/NameProfile.tsx:19]: it's not used anywhere so far )

File: .gitignore (hunk 1 above)
Comment (LyubovSugak) [reply .gitignore:28]: my bad, removed it

File: src/components/firstProfile/NameProfile.tsx (hunk 2 above)
Comment (Svitlana-G) [reply src/components/firstProfile/NameProfile.tsx:19]: Do we need to `setFullName`, if `fullName` isn't used?

Comment (LyubovSugak) [reply src/components/firstProfile/NameProfile.tsx:19]: Let's leave it as it is, bc it requires fixes anyway (i approved it to let Sofia moving further)

Comment (Svitlana-G) [reply src/components/firstProfile/NameProfile.tsx:19]: Ok

**Review by Svitlana-G — APPROVED** [review]
